docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/upgrade-to-apollov4.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Apollo Server v3 upgraded to Apollo Server v4
+description: The upgrade from Apollo Server v3 to v4 and graphql ^15 to ^16.
+sidebar_label: Upgrade to Apollo Server v4
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - Content API
+ - GraphQL
+ - GraphQL API
+---
+
+import FeedbackCallout from '/docs/snippets/backend-customization-feedback-cta.md'
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Apollo Server v3 upgraded to Apollo Server v4
+
+Strapi 5 has some changes for user code that relied on specific versions.
+<Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Apollo Server v3 for the GraphQL server and graphql ^15 for the GraphQL module.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi 5**
+
+Apollo Server v4 for the GraphQL server and graphql ^16 for the GraphQL module.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+:::callout
+Strapi automatically sets `status400ForVariableCoercionErrors: true` in the Apollo Server configuration due to a breaking change introduced by Apollo in v4. This behavior will be the default again in Apollo v5.
+:::
+
+:::callout
+We upgraded the package graphql from ^15 to ^16, and only one version can be included in any project. Therefore, user code or plugins relying on graphql:15 need to use the same version range as Strapi.
+:::
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+- The migration process for user code and configuration is too complicated for Strapi to cover completely. 
```

Comment:
Please _never_ write something is easy or difficult. This is left to the users' appreciation. See our 12 rules in the [handbook](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing).

Rewrite this with a different tone, something like: "For detailed instructions on how to migrate to Apollo v4, please refer to Apollo official docs (+ add external link here)."